draft-ietf-add-ddr.md

```diff
@@ -388,7 +388,7 @@ where the Unencrypted Resolver and Designated Resolver have the same IP address.
 
 The constraints on validation of Designated Resolvers specified here apply specifically
 to the automatic discovery mechanisms defined in this document, which are
-referred to as Authenticated Discovery and Opportunistic Discovery. Clients
+referred to as Verified Discovery and Opportunistic Discovery. Clients
 MAY use some other mechanism to validate and use Designated Resolvers discovered
```

Comment:
Where this is talking about a mechanism used in lieu of DDR verification, maybe it should also use the word "verify"? Not all existing uses of "validate" match that expectation, but this paragraph's instances seem to do so.

Reply:
Good call, fixed